provider/README.md

```diff
@@ -336,8 +336,51 @@ When multiple query parameters are specified, they should all apply to the retur
 [Top][toc]
 
 ## Realtime Data
+All providers should provide an endpoint that displays the current status of a providers vehicles on the PROW. Only vehicles that are currently in an available, unavailable, or reserved states should be returned in this payload. Data in this endpoint should generally reconcile with data from the status changes enpdoint.
 
-All MDS compatible `provider` APIs must expose a public [GBFS](https://github.com/NABSA/gbfs) feed as well. Given that GBFS hasn't fully [evolved to support dockless mobility](https://github.com/NABSA/gbfs/pull/92) yet, we follow the current guidelines in making bike information avaliable to the public. 
+### Vehicles
```

Comment:
Would it make sense to follow REST-like practice of returning a single vehicles if GET `/vehicles/{device_id}` is called?

Reply:
[Decided not to support REST-like querying for now](https://github.com/openmobilityfoundation/mobility-data-specification/pull/376#issuecomment-575301414)